app/controllers/staff/base_controller.rb

```diff
@@ -1,12 +1,22 @@
+class UserNotAuthorised < StandardError; end
```

Comment:
Base controller is not becoming quite heavy with auth logic. Should we move it to an authentication concern?

Reply:
I think yes